batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/Utils/Utilities.java

```diff
@@ -0,0 +1,33 @@
+package com.lftechnology.batch7crud.Utils;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * Created by leapfrog on 1/27/16.
+ */
+public class Utilities {
+  private Utilities(){
+
+  }
+
+  public static String getMD5(String input) throws DataException {
+    try {
+      MessageDigest md = MessageDigest.getInstance("MD5");
+      byte[] messageDigest = md.digest(input.getBytes());
+      BigInteger number = new BigInteger(1, messageDigest);
+      String hashText = number.toString(16);
+      // Now we need to zero pad it if you actually want the full 32 chars.
+      while (hashText.length() < 32) {
+        hashText = "0" + hashText;
+      }
+      return hashText;
+    }
+    catch (NoSuchAlgorithmException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)